packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -168,10 +172,32 @@ export default function MainFeedLayout({
     getFeatureValue(Features.FeedVersion, flags),
     10,
   );
+  const myFeedOnboardingVersion = getFeatureValue(
+    Features.MyFeedOnboardingVersion,
+    flags,
+  );
   const feedName = feedNameProp === 'default' ? defaultFeed : feedNameProp;
   const isMyFeed = feedName === 'my-feed';
   const propsByFeed = getPropsByFeed({ shouldShowMyFeed });
 
+  const [isFirstSession, setIsFirstSession, isSessionLoaded] =
+    usePersistentContext(FIRST_TIME_SESSION, isFirstVisit);
+
+  useEffect(() => {
+    if (user) {
+      setCreateMyFeed(false);
```

Comment:
Would be nice to have batching here - but we need to update React version (we can go with unstable_batchedUpdates from core React API, but I don't really like it even though it's pretty stable)

Reply:
React version is a limitation of the extension at this point.